### ISSUE TITLE
No games in grid view. 

### ISSUE THREAD

**Issue by ShadoUrufu** [JosefNemec/Playnite#2474]
[Image](https://cdn.discordapp.com/attachments/352590803488604171/885744654404960376/unknown.png)

I have no clue what i messed around with, pressed a few things here and there while looking at a game. [like humble or arcade links] 

At first, i only saw the humble games, even in the -all- view. and now, they're all completely gone. I can still see them in fullscreen view however. And i can still access them by other means through the app. 

This is only an issue with grid view in desktop mode.

**Comment by darklinkpower** [comment]
Where is all the information required in the issue template? Please don't just delete it and provide all the requested information.

It also seems that you are using a custom theme. I suggest checking with the default theme since some themes can introduce issues.

**Comment by ShadoUrufu** [comment]
[Image](https://cdn.discordapp.com/attachments/352590803488604171/885751285419212840/unknown.png)
This is with the classic plain theme. The theme was working before, as i was using it to sort out the library. 

The filter is completely disabled. 

The template didn't really matter for this. Here's the ID: c7bf26f3-078a-45ff-b576-3df9e5e5dc80

I don't even know how I produced this, it just happened while I was sorting my library.

**Comment by JosefNemec** [comment]
Can you try in safe mode? You can restart to it from from Help main menu.

**Comment by JosefNemec** [comment]
> The template didn't really matter for this.

Based on what? There's a reason the template is there, especially data from diagnostics package are very valuable.

**Comment by ShadoUrufu** [comment]
> Can you try in safe mode? You can restart to it from from Help main menu.

[Image](https://cdn.discordapp.com/attachments/352590803488604171/885775032922562570/unknown.png)

I've tried with plugins, i've tried without. Here's the safe mode view. I haven't changed anything from the last pic, other than restarting in safe mode. 

And to clarify, I pressed the 'humble bundle' link listed on one of the games, thinking it would open the website. Might of presed it several times, I don't remember. Did some things, app restarted, it only showed humble stuff. 

I tried it again, pressed a few things, and then the humble stuff vanished. finally noticed that clicking them checked the filters on for that stuff, so i made sure to turn it off. and still empty.

**Comment by ShadoUrufu** [comment]
> > The template didn't really matter for this.
> 
> Based on what? There's a reason the template is there, especially data from diagnostics package are very valuable.

based on what i read from it. didn't know how to get the package ID stuff. And I'm just trying to figure out what I did wrong, what keys I might of hit, since I move my keyboard around a lot, or just what could of happened. 

Something happened, not sure what, figured it might just be something I did and not really a bug. It's also rather late for me, so I tend not to think too clearly. 

Anyway, here's a list of all the extensions I added, all of them were installed today. 
Theme is eMixedNiteMC with Lato-Edited font installed. will try changing that. 

[Image](https://cdn.discordapp.com/attachments/352590803488604171/885777070947446794/unknown.png)
